test-app/tests/integration/components/headless-form-data-test.gts

```diff
@@ -30,6 +30,7 @@ module('Integration Component HeadlessForm > Data', function (hooks) {
         country: 'USA',
         comments: 'lorem ipsum',
         acceptTerms: true,
+        age: 21,
```

Comment:
I wasn't entirely sure about the correct value here, but I figured it should be just about right!? 🙃

Reply:
😂 😂 😂 I wish!!! 😂 😂 😂